Runtime/Game/Requests/LootLockerLeaderboardDetails.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Net;
+using LootLocker.Requests;
+using UnityEngine;
+
+namespace LootLocker.Requests
+{
+    public class LootLockerLeaderboardDetailResponse : LootLockerResponse
+    {
+        public string created_at { get; set; }
+        public string updated_at { get; set; }
+        public string key { get; set; }
+        public string direction_method { get; set; }
+        public string name { get; set; }
+        public string type { get; set; }
+        public int id { get; set; }
+        public int game_id { get; set; }
```

Comment:
I think you've been looking at the admin api version of this api. This one shouldn't have (at the very least, could be more fields that shouldn't be here): game_id, enable_game_api_writes, and possibly overwrite_score_on_submit?

Reply:
Ben says it's the same for the game api :) Should we discuss if this should be there?